frontend/src/components/Layouts/HomePageComponent/components/CategoryOutput/CategoryOutput.jsx

```diff
@@ -56,24 +56,27 @@ const CategoryOutput = ({ title, categoryId }) => {
         <p>Loading...</p>
       ) : (
         <div className={styles.categoryOutput}>
-          <ul className={styles.cardOutput}>
+          <ul
+            className={`${styles.cardOutput} ${
+              title === 'Акційна ціна' ? styles.discounted : ''
+            }`}
+          >
             {data &&
               data.map((card) => (
-                <li key={card.id} className={styles.card}>
```

Comment:
You deleted tag `<li>` Do you need tag `<ul>` after that?

Reply:
Yes, inner component has li tag. I did it in purpose to simplify code and its manipulation

Reply:
I have already found `li` in component Card